tuna/fin_builder.py

```diff
@@ -91,6 +91,13 @@ def process_pdb_compile(self, session, fin_json):
 
   def step(self):
     """Main functionality of the builder class. It picks up jobs in new state and compiles them"""
+
+    try:
+      self.check_env()
+    except ValueError as verr:
+      self.logger.error(verr)
+      return False
```

Comment:
This will add extra file I/O for each job. Is there a clean way to check once before we launch the worker?

Reply:
ill investigate moving this call somewhere else

Reply:
Perhaps in the constructer for the Fin class objects? Or we could store state here to only check once during a run?